rest-api/config/config_prod.json

```diff
@@ -80,8 +80,10 @@
       "awardee": "TRANS_AM",
       "whitelisted_ip_ranges": {
         "ip4": [
-          "66.189.127/27",
-          "68.184.42.162/27"
+          "161.249.83.116",
+          "161.249.147.132",
+          "161.249.241.64/26",
+          "161.249.150.128/26"
         ]
       },
       "description": "Using subnet mask these ranges cover 66.189.127.0-32 and 68.184.42.162.160-191"
```

Comment:
please update descriptions. Also, you may want to double check with Celecia that these IP's would apply to prod and stable. Usually different ranges are used for each env. This may only be a prod change (assuming from the urgency of the ticket)